ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c

```diff
@@ -192,6 +134,12 @@ nnfw_open (const GstTensorFilterProperties * prop, void **private_data)
     }
   }
 
+  /* validate model file first */
+  if (!g_file_test (prop->model_files[0], G_FILE_TEST_EXISTS)) {
```

Comment:
when the input is a model_path, how will this work?

Reply:
how about move this check to line 160 inside the else condition?

Reply:
https://developer.gnome.org/glib/stable/glib-File-Utilities.html
G_FILE_TEST_EXISTS returns TRUE if the file exists. It may or may not be a regular file.